### ISSUE TITLE
License for the ferrari.ttf font

### ISSUE THREAD

**Issue by fpesari** [EmissionControl2/EmissionControl2#104]
Hello,

what is the license for the [ferrari.ttf](https://github.com/EmissionControl2/EmissionControl2/blob/master/externalResources/fonts/ferrari.ttf) font? Since it is inside the `externalResources` directory, I assume it was not made from scratch. I tried doing some web searches but I could not find this exact font.

If it's not released under a suitable free license (es. the Apache License 2.0`*` adopted by Roboto, the other included font), maybe it's better to replace it with another font, so that packagers who have to adhere to specific guidelines don't have licensing problems which EmissionControl2 normally doesn't have, being released under the GPLv3. Thank you, from a packager :)

`*` = Which by the way needs attribution, so you have to include the license notice for it since it is bundled with EmissionControl2

**Comment by rodneydup** [comment]
Curtis wanted us to imitate the "Engine Start" button on Ferrari cars lol... I had played with FontForge a bit and so I tried my best to roughly mimic that font.

Thank you for pointing this out. It reminded me that I wanted to change the way fonts are handled in EmissionControl2. As of the latest commit, the app will no longer distribute font "ttf" files. The fonts are now bundled into the binary code (with a tiny increase in the file size).

I still want to give proper attribution for Roboto. I created a notice.txt in the repository that hopefully covers this.
